cupy/cuda/cudnn.pyx

```diff
@@ -995,6 +995,8 @@ cpdef poolingBackward(
 # Batch Normalization
 ###############################################################################
 
+CUDNN_BN_MIN_EPSILON = 1e-5
```

Comment:
How about fixing `cudnn.pxd` file?

Reply:
Initially I tried the following code in pxd, but it does not work.

```py
cpdef double CUDNN_BN_MIN_EPSILON = 1e-5
```
Also, it cannot be enum as it is not integer.

Reply:
OK.